### ISSUE TITLE
TOC keyboard navigation

### ISSUE THREAD

**Issue by greatislander** [pressbooks/pressbooks-book#15]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing this issue with a book using this theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Description

The tables of contents in online books that have more than one page within chapters cannot be navigated with a keyboard. In the table of contents, the individual sections are collapsed into their containing chapters and there is no way to expand those sections with a keyboard -- only a mouse. You can tab through the links to pages within chapters that are already expanded but the keyboard skips over any chapters that are not. This is a huge barrier to accessibility for anyone who uses a screen reader and relies on keyboard navigation.

See https://github.com/pressbooks/pressbooks/issues/526

### Steps to Reproduce

1. Visit a webbook and click "Read".
2. Attempt to use the keyboard to navigate into the table of contents.

**Expected behavior:** Keyboard should be able to navigate through all sub-chapters.

**Actual behavior:** Keyboard cannot navigate into collapsed sub-chapters when the TOC is more than a page long.
### System Information

Book Theme: Pressbooks Book
Pressbooks: 4.0.0-dev
WordPress: 4.8.0
PHP: 7.1
